.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,13 @@
+
+# API Muncher
+Congratulations! You're submitting your assignment!
+
+## Comprehension Questions
+| Question | Answer    |
+| :------------- | :------------- |
+| How did you go about exploring the Edamam API, how did you try querying the API? |  |
+| Describe your API Wrapper.  How did you decide on the methods you created? |  |
+| Describe how you approached testing your API Wrapper, what edge cases do you take? |  |
```

Comment:
Maybe this could be simplified to "Describe an edge case or failure case test you wrote for your API Wrapper"